### PR TITLE
Add Labs Footer and Labs Label components

### PR DESCRIPTION
**Pull request by oelhanafey**
[US137268](https://rally1.rallydev.com/#/?detail=/userstory/630020757369&fdp=true) 
The rally ticket contains the design docs for this story.

This PR adds the labs label and labs footer components. `<d2l-daylight-labs-label>` is shown next to the the title of a labs page and displays the labs icon and the text "Labs" so that users know that they are viewing a labs component. This needed to be split into a separate component because adding it directly to the page layout resulted in the "Labs" text being added to the page title when searching. (e.g. Accordion became AccordionLabs)

`<d2l-daylight-labs-footer>` is a footer that sticks to the bottom of labs pages and informs the user that labs components may not be as complete as our core components as well as linking to more information about labs components.

I also made a change to the labs issues by adding the `labs` boolean attribute to the front matter of labs pages. This makes it easier to check if a page is a labs page (and easier to read the code).

### CODE REVIEW

File: pages/_components/labs-footer.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+			}
+			a {
+				color: white;
+				grid-column-end: 3;
+				grid-column-start: 2;
+				text-decoration: none;
+			}
+			.banner-grid {
+				display: grid;
+				grid-template-columns: minmax(14.8rem, 20rem) minmax(calc(284px + 2*2.7rem), auto);
```
Comment (oelhanafey) [inline pages/_components/labs-footer.js:21]: This is the same grid dimensions that the main sidebar and content panels have. I wasn't able to get the positioning to work properly adding the banner directly to the content panel so I simulated it by creating a new grid with identical dimensions.

File: pages/_components/labs-footer.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+				justify-content: center;
+				padding: 0.9rem;
+				text-align: center;
+			}
+			.banner-inner {
+				-moz-box-orient: vertical;
+				-webkit-box-orient: vertical;
+				display: -webkit-inline-box;
+				-webkit-line-clamp: 1;
+				overflow: hidden;
```
Comment (oelhanafey) [inline pages/_components/labs-footer.js:35]: Despite the name, firefox does support `-webkit-inline-box` and `-webkit-line-clamp`: https://caniuse.com/?search=line-clamp

File: pages/_components/labs-label.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+				color: var(--d2l-color-green-70);
+				margin-left: 0.3rem;
+			}
+		`];
+	}
+
+	render() {
+		return html`
+			<div class="d2l-label-container">
+				<img class="d2l-label-img" src="/img/labs-icon.svg" alt="labs icon" />
```
Comment (dlockhart) [inline pages/_components/labs-label.js:31]: Ditto here -- no `alt` needed and suggest setting width & height.

File: pages/_components/labs-label.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+			.d2l-label-text {
+				color: var(--d2l-color-green-70);
+				margin-left: 0.3rem;
+			}
+		`];
+	}
+
+	render() {
+		return html`
+			<div class="d2l-label-container">
```
Comment (dlockhart) [inline pages/_components/labs-label.js:30]: Is this extra container needed, or could its styles just be applied to the host?

File: pages/_components/labs-label.js [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+				<img class="d2l-label-img" src="/img/labs-icon.svg" alt="labs icon" />
+				<span class="d2l-labs-label d2l-label-text">Labs</span>
```
[the remark this answers is not in the document]
Comment (oelhanafey) [reply pages/_components/labs-label.js]: Oops, I meant for my CSS to affect that class instead of altering the `d2l-label-text` typography style.

**Review by dlockhart — APPROVED** [review]
LGTM!

**Pull request merged**